load.php

```diff
@@ -248,6 +255,19 @@ function perflab_can_load_module( $module ) {
 	return (bool) $module();
 }
 
+/**
+ * Gets the standalone plugin configuration.
+ *
+ * @since n.e.x.t
+ *
+ * @return array Array of standalone plugin configuration.
+ */
+function perflab_get_standalone_plugins_config() {
```

Comment:
Maybe we can clarify what this is for?
```suggestion
/**
 * Gets the standalone plugin constants used for each module / plugin.
 *
 * @since n.e.x.t
 *
 * @return array Map of module path to version constant used.
 */
function perflab_get_standalone_plugins_constants() {
```